lib/cli.js

```diff
@@ -326,7 +326,7 @@ const meowOptions = {
  * @param {string[]} argv
  * @returns {Promise<any>}
  */
-module.exports = (argv) => {
+module.exports = async (argv) => {
```

Comment:
[suggestion] How about changing to a named function like other modules?

```suggestion
module.exports = async function cli(argv) {
```

Reply:
I can't do that because `cli` variable declared right on the next line and used across whole function :)

Reply:
Thanks! I see. 😃